### ISSUE TITLE
Updating a document's content in the browser clobbers its permissions

### ISSUE THREAD

**Issue by mikrovvelle** [fgeorges/expath-ml-console#7]
Steps to reproduce:

1. insert an XML document to the database (outside of expath, if necessary), add at least 1 permission
1. navigate to the document's location using expath (logged in as ML admin)
1. edit the document content and save it using the built-in editor
1. any permissions added in the first step are missing.

I would expect the permissions to be unchanged, but it looks like they're getting clobbered by the admin's default permissions (which are none).

Tested on current master branch, safari, ml 9.0-1.1. Observed earlier on ml8 as well.

**Comment by fgeorges** [comment]
Hi @mikrovvelle, I am so glad to ee you here.  For some reason, I missed your message at the time, sorry about that :-(

I cannot reproduce this behaviour.  Which version are you using?  I know this is not obvious, but `master` is very old.  You'd better use `develop`, or even better now, `project-environs` (and yes, I should really merge them back to `master`).

The way I tried to reproduce it:

- I created a new database `test`
- went to http://localhost:8010/db/test/roots
- `Add files...` > `Create document` > `/hello.xml` > `Create`
- added a collection to make sure they are kept as well
- added a couple of random permissions
- changed the content of the file and pressed `Save`
- reloaded the page after confirmation, to be sure I see latest info

Then I can see the collection and permissions are preserved.
